eZ/Publish/Core/REST/Client/UserService.php

```diff
@@ -104,13 +103,15 @@ public function createUserGroup(UserGroupCreateStruct $userGroupCreateStruct, Us
      * Loads a user group for the given id.
      *
      * @param mixed $id
+     * @param string[] $prioritizedLanguages Used as prioritized language code on translated properties of returned object.
      *
      * @return \eZ\Publish\API\Repository\Values\User\UserGroup
      *
+     * @throws \Exception Method is not implemented
```

Comment:
we don't normally do this, besides as the other changes involved move to just inherit doc